index.html

```diff
@@ -1342,10 +1342,10 @@ <h4>skolemizeExpandedJsonLd</h4>
 
           <p>
 The following algorithm replaces all blank node identifiers in an expanded
-JSON-LD document with custom-scheme URNs. The required inputs are an expanded
-JSON-LD document (<var>expanded</var>), a custom URN scheme
+JSON-LD document with custom-scheme URNs. The required inputs are an expanded JSON-LD document
```

Comment:
@Wind4Greg, does this work?

```suggestion
JSON-LD document with custom-scheme URNs, including assigning such URNs to
blank nodes that are unlabeled. The required inputs are an expanded JSON-LD document
```

Reply:
Excellent @dlongley that says it all!